src/sensor_status_t.jl

```diff
@@ -0,0 +1,13 @@
+# The sensor_status_t has a field named `type`, which is a reserved
+# keyword in Julia v0.7 and below. To get around that, we can just
+# construct the expression directly instead of relying on the parser.
```

Comment:
That's unfortunate. Nice job working around this limitation.

Reply:
Heh, yeah. It's actually not so bad (I can't imagine trying to do this in Python if we had an lcmtype with `class` as a field...).

Reply:
Plus, in v1.0 we can get rid of this entirely!